CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [FEATURE] Add support for zone aware reads. #3414
```

Comment:
We usually consider feature a branch new feature. In this case we're improving an existing one.

```suggestion
* [ENHANCEMENT] Added zone-awareness support on queries. When zone-awareness is enabled, queries will still succeed if all ingesters in a single zone will fail. #3414
```

Reply:
Applied directly.